test/integration/parachain/staging/sequential/replace.test.ts

```diff
@@ -103,6 +103,7 @@ describe("replace", () => {
                     interBtcAPI.replace.request(replaceAmount, collateralCurrency),
                     false
                 );
+                console.log("\n\n res, val", result.status.type, result.status.asFinalized);
```

Comment:
Is this log still needed? Looks like it might be a leftover from early trials/testing.

Reply:
Not needed, good eyes!